modules/apps/object/object-web/src/main/resources/META-INF/resources/js/index.d.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+interface IObjectFieldBusinessType {
+	businessType: string;
+	dbType: string;
+	description: string;
+	label: string;
+}
```

Comment:
You did great! I only found a possible improvement.

To reuse the `IObjectFieldBusinessType` interface it is better to put it inside `js/modules/global.d.ts` instead of creating an index.d.ts just for this case